includes/class-abandoned-payment-notification.php

```diff
@@ -42,6 +42,9 @@ public function send_abandoned_payment_notifications() {
             }
 
             $payment = WPBDP_Payment::objects()->get( $data->id );
+			if ( !$payment ) {
```

Comment:
The phpcs here is still wrong...

Reply:
I'll need to correct PHPCS formatting for the entire file. As it uses spaces as opposed to tabs

Reply:
Unless I change to spaces from tabs, as most of the classes and files uses spaces as opposed to tabs

Reply:
The tabs here are fine. The problem is "!$payment". This is the same issue we've been talking about here:
https://github.com/Strategy11/business-directory-plugin/pull/55#discussion_r731130874